cmd/hz/app/app.go

```diff
@@ -184,7 +184,7 @@ func Init() *cli.App {
 	unsetOmitemptyFlag := cli.BoolFlag{Name: "unset_omitempty", Usage: "Remove 'omitempty' tag for generated struct.", Destination: &globalArgs.UnsetOmitempty}
 	protoCamelJSONTag := cli.BoolFlag{Name: "pb_camel_json_tag", Usage: "Convert Name style for json tag to camel(Only works protobuf).", Destination: &globalArgs.ProtobufCamelJSONTag}
 	snakeNameFlag := cli.BoolFlag{Name: "snake_tag", Usage: "Use snake_case style naming for tags. (Only works for 'form', 'query', 'json')", Destination: &globalArgs.SnakeName}
-	rmTagFlag := cli.StringSliceFlag{Name: "rm_tag", Usage: "Remove the specified tag"}
+	rmTagFlag := cli.StringSliceFlag{Name: "rm_tag", Usage: "Remove the specified tag, it's priority is lower than the annotation."}
```

Comment:
A little bit confusing.
How about: "Remove the default tags(xxx/xxx/xxx). If the annotation tag is set explicitly, it will not be deleted. In this case, you need to explicitly remove it from your idl." ？